octo/contribution.go

```diff
@@ -37,14 +37,21 @@ type Contribution struct {
 	Permissions os.FileMode
 	Structure   gotree.Tree
 	Content     []byte
+	Namespace   string
 }
 
-func NewActionContribution(workflow actions.Workflow) (Contribution, error) {
+func NewActionContributionWithNamespace(namespace string, workflow actions.Workflow) (Contribution, error) {
 	var (
 		c   Contribution
 		err error
 	)
-	c.Path = filepath.Join(".github", "workflows", fmt.Sprintf("%s.yml", strcase.ToKebab(workflow.Name)))
+
+	workflowName := fmt.Sprintf("%s-%s.yml", namespace, strcase.ToKebab(workflow.Name))
+	if namespace == "" {
+		workflowName = fmt.Sprintf("%s.yml", strcase.ToKebab(workflow.Name))
+	}
```

Comment:
Not sure if this is better but it just reads a little weird at the moment?
```suggestion
	var workflowName string
	if namespace == "" {
		workflowName = fmt.Sprintf("%s.yml", strcase.ToKebab(workflow.Name))
	} else {
		workflowName = fmt.Sprintf("%s-%s.yml", namespace, strcase.ToKebab(workflow.Name))
	}
```